lib/src/center_play_button.dart

```diff
@@ -21,23 +21,23 @@ class CenterPlayButton extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Container(
+    return ColoredBox(
       color: Colors.transparent,
       child: Center(
-        child: AnimatedOpacity(
-          opacity: show ? 1.0 : 0.0,
-          duration: const Duration(milliseconds: 300),
-          child: Container(
-            decoration: BoxDecoration(
-              color: backgroundColor,
-              shape: BoxShape.circle,
-            ),
-            child: Padding(
-              padding: const EdgeInsets.all(12.0),
+        child: UnconstrainedBox(
+          child: AnimatedOpacity(
+            opacity: show ? 1.0 : 0.0,
+            duration: const Duration(milliseconds: 300),
+            child: DecoratedBox(
+              decoration: BoxDecoration(
+                color: backgroundColor,
+                shape: BoxShape.circle,
+              ),
```

Comment:
@luis901101 Thanks for the PR. Quick question. Does this change in any way break non-Material 3 setups?

For example, if the user wants to not use Material 3, does this change create an impact?

If so, you may need to only apply this if the current `Theme` has Material 3 enabled, so that it doesn't break any users who don't want to opt-in to that at the moment.

Reply:
Hi, I tested both cases, Material3 and Non-Material3 themes and it works as expected in both cases.